CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+* Update docs to reflect OpenSSL version requirements
```

Comment:
I left a comment on the pull request, did you see it? https://github.com/boostorg/beast/pull/1928#issuecomment-621901976